OpenKeychain/src/main/java/org/sufficientlysecure/keychain/keyimport/Keyserver.java

```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright (C) 2012-2014 Dominik Schürmann <dominik@dominikschuermann.de>
- * Copyright (C) 2011-2014 Thialfihar <thi@thialfihar.org>
- * Copyright (C) 2011 Senecaso
+ * Copyright (C) 2012-2015 Dominik Schürmann <dominik@dominikschuermann.de>
+ * Copyright (C) 2011-2015 Thialfihar <thi@thialfihar.org>
+ * Copyright (C) 2014-2015 Senecaso
```

Comment:
Good catch. Didn't update the single year replacement when I generalized it to years other than 2014.